plugins/source/aws/resources/services/iam/policy_services_last_accessed_fetch..go

```diff
@@ -0,0 +1,38 @@
+package iam
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/iam/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"golang.org/x/sync/errgroup"
+)
+
+func fetchIamPolicyServicesLastAccessed(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- interface{}) error {
+	c := meta.(*client.Client)
+	svc := c.Services().Iam
+	ch := make(chan interface{})
+
+	g, ctx := errgroup.WithContext(ctx)
+	g.Go(func() error {
+		if err := fetchIamPolicies(ctx, meta, nil, ch); err != nil {
+			return err
```

Comment:
If this returns an error, I think we should still close the channel, otherwise the goroutine below will hang. (Same comment applies to the other similar functions)